libs/config/src/wire/v6.rs

```diff
@@ -43,14 +43,95 @@ pub struct Net {
     pub authoritative: bool,
 }
 
-// TODO allow configuring server id
-// #[derive(Serialize, Deserialize, Debug, PartialEq, Eq, Clone)]
-// #[serde(tag = "type", content = "value", rename_all = "snake_case")]
-// pub enum ServerDuid {
-//     LLT(String),
-//     LL(String),
-//     EN(String),
-// }
+#[derive(Serialize, Deserialize, Debug, PartialEq, Eq, Clone)]
+pub enum DuidType {
+    LLT,
+    LL,
+    EN,
+    UUID,
+}
+
+impl Default for DuidType {
+    fn default() -> Self {
+        Self::LLT
+    }
+}
+
+#[derive(Serialize, Deserialize, Debug, PartialEq, Eq, Clone)]
+#[serde(tag = "duid_type")]
+pub enum ServerDuidInfo {
```

Comment:
This name is kind of weird... any better idea?🤔

Reply:
I'm not the best namer either :smile: